pubsub/api/test/pubsubTest.php

```diff
@@ -195,6 +195,30 @@ public function testCreateAndDeleteSubscription()
         $this->assertRegExp(sprintf('/%s/', $subscription), $output);
     }
 
+    public function testCreateAndDeleteSubscriptionWithFilter()
+    {
+        $topic = $this->requireEnv('GOOGLE_PUBSUB_TOPIC');
+        $subscription = 'test-subscription-' . rand();
+        $filter = 'attributes.author="unknown"';
+        $output = $this->runFunctionSnippet('create_subscription_with_filter', [
+            self::$projectId,
+            $topic,
+            $subscription,
+            $filter
+        ]);
+        $this->assertStringContainsString('Subscription created:', $output);
+        $this->assertStringContainsString(sprintf('%s', $subscription), $output);
```

Comment:
Why have two assertions here?
```suggestion
        $this->assertStringContainsString(sprintf('Subscription created: %s', $subscription), $output);
```